ffwd-client/src/main/java/com/spotify/ffwd/FastForward.java

```diff
@@ -31,10 +31,31 @@
 
 public class FastForward {
 
+  /**
+   * If you are using older version, please use Version.V0.
```

Comment:
* If you are using an older version, please use Version.V0.
   * This variable will eventually be removed.